doc/deprecated.md

```diff
@@ -13,7 +13,8 @@ longer. In particular:
 A list of deprecated features and the version in which they were deprecated follows:
 
 -   Support to NGSI v1.
--   Support to Node.js v4 in iotagent-node-lib 2.8.1. The use of Node.js v10 is highly recommended.
+-   Support to Node.js v4 in iotagent-node-lib 2.8.1.
+-   Support to Node.js v10 in iotagent-ul 2.15.0. The use of Node.js v12 is highly recommended.
```

Comment:
For the sake of completeness, maybe information about versions between v4 and v10 should be included (it should be easy to know looking at https://github.com/telefonicaid/iotagent-node-lib/releases )... although maybe it doesn't worth the time.

Not sure how it's better...

Reply:
Fixed 9b5f768

Reply:
Fixed 479f120